modules/ossm-scoping-sm-discovery-selectors-istio-ambient-mode.adoc

```diff
@@ -0,0 +1,69 @@
+// Module included in the following assemblies:
+
+// * service-mesh-docs-main/install/ossm-istio-ambient-mode.adoc
+
+:_mod-docs-content-type: Procedure
+[id="ossm-scoping-sm-discovery-selectors-istio-ambient-mode_{context}"]
+= Scoping the Service Mesh with discovery selectors in Istio ambient mode
+
+To limit the scope of the {SMProduct} in {istio} ambient mode, you can configure `discoverySelectors` in the `meshConfig` section of the `{istio}` resource. The configuration controls which namespaces the control plane discovers based on label selectors.
+
+.Prerequisites
+
+* You have deployed a cluster on {ocp-product-title} 4.19 or later.
+* You have created an `Istio` control plane resource.
+* You have created an `IstioCNI` resource.
+* You have created a `Ztunnel` resource.
+
+.Procedure
+
+. Add a label to the namespace containing the `{istio}` control plane resource, for example, the `istio-system` namespace.
```

Comment:
These steps with commands need to have something like "by running the following command" somewhere in them. I am not sure what would be a good way to add that here since the sentence is already kind of long. Maybe this would still be short enough:
```suggestion
. Add a label to the namespace containing the `{istio}` control plane resource, for example, the `istio-system` namespace, by running the following command:
```

Please apply this feedback to the next two steps as well